FirstTask.py

```diff
@@ -0,0 +1,230 @@
+        
+        elif option == 2:
+            if bank.is_logged_in() == False:
+                account_number = int(input("Enter Account Number: "))
+                code = int(input("Enter Passcode: "))
+                status, response = bank.login(account_number, code)
+
+            if bank.is_logged_in():
+                print("Login Successful!")
+                input("\nPress Any Key to Continue\n")
+                select = 1
+                while not select == 0:
+                    sys('clear')
+                    select = int(input("1) Check Balance \n2) Deposit Bill \n3) Draw Money \n4) Transfer Money \n5) Account Type \n6) Compare Your Account \n7) Account Details \n0) Log Out \nChoose:"))
+                    if select == 1:
+                        print(bank.logged_in_account.check_balance())
+                        input("\nPress Any Key to Continue\n")
+
+                    elif select == 2:
+                        deposit_money = int(input("Enter Deposit Amount: "))
+                        if bank.logged_in_account.deposit_bill(deposit_money):
+                            print("Money Successfully Deposited")
+                        else:
+                            print("Money must be Greater than 0!")
+                        input("\nPress Any Key to Continue\n")
+
+                    elif select == 3:
+                        amount = int(input("Enter Amount to withdraw: "))
+                        if bank.logged_in_account.draw_money(amount):
+                            print("Money Draw Successfull")
+                        else:
+                            print("Not Enough Cash")
+                        input("\nPress Any Key to Continue\n")
+                    
+                    elif select == 4:
+                        money = int(input("Enter Transfer Amount: "))
+                        receiver_account_number = int(input("Enter Account Number of Other User: "))
+                        status, response = bank.transfer_money(bank.logged_in_account.account_number(), receiver_account_number, money)
+                        if status:
+                            print(response)
+                        else:
+                            print(response)
+                        input("\nPress Any Key to Continue\n")
+                            
+                    elif select == 5:
+                        print(bank.logged_in_account.check_account_type())
+                        input("\nPress Any Key to Continue\n")
+                    
+                    elif select == 6:
+                        other_account_number = int(input("Enter Account Number you want to compare with: "))
+                        status, response = bank.compare(bank.logged_in_account.account_number(), other_account_number)
+                        if(status):
+                            print(response)
+                        else:
+                            print("Invalid Account!")

```

Comment:
We can just print the response without any check here as well because we need to also show the if he entered a wrong account number to compare with.